src/main/resources/api/data-repository-openapi.yaml

```
@@ -7445,6 +7445,12 @@ components:
           type: string
         updatedDate:
           type: string
+        jobId:
+          type: string
+          description: The id for the job to create the snapshot once approved
```

I'm not sure if we've been consistent in using this schema, but I think we should use the flight id schema where possible.
```suggestion
        jobId:
          $ref: '#/components/schemas/ShortIdProperty'
```